src/pages/vaults/VaultPositionsTable.tsx

```diff
@@ -51,20 +52,28 @@ export const VaultPositionsTable = ({ className }: { className?: string }) => {
           label: stringGetter({ key: STRING_KEYS.MARKET }),
           renderCell: ({ marketId, currentLeverageMultiple }) => {
             const asset = marketId != null ? marketIdToAssetMetadataMap[marketId] : undefined;
+
             return (
               // eslint-disable-next-line jsx-a11y/interactive-supports-focus
               <div
                 tw="cursor-pointer rounded-0.5 hover:bg-color-layer-3"
                 role="button"
                 onClick={() =>
+                  marketId != null &&
+                  marketsData[marketId] != null &&
                   navigate(`${AppRoute.Trade}/${marketId}`, { state: { from: AppRoute.Vault } })
                 }
               >
                 <TableCell
                   stacked
-                  slotLeft={<AssetIcon symbol={asset?.id} tw="[--asset-icon-size:2.5em]" />}
+                  slotLeft={
+                    <AssetIcon
+                      symbol={marketId === USDC_MARKET_HARDCODED ? 'USDC' : asset?.id}
```

Comment:
maybe jsut write a helper function that converts marketId to the symbol used for icons? and then u can add a comment to explain why this usdc thing is special cased

Reply:
It's only here so I don't think it's really worth the function but I added a nice comment at the const declaration to explain what's happening.